.github/labeler.yml

```
@@ -29,3 +29,11 @@ trino:
 bigquery:
 - changed-files:
   - any-glob-to-any-file: ['**/*bigquery*']
+
+ci:
+- changed-files:
+  - any-glob-to-any-file: ['.github/**']
+
+v1-engine-changed:
+- changed-files:
+  - any-glob-to-any-file: ['wren-base/**', 'wren-main/**', 'wren-server/**', 'wren-tests/**']
```

```suggestion
  - any-glob-to-any-file: ['wren-base/**', 'wren-main/**', 'wren-server/**', 'wren-tests/**', 'trino-parser/**', 'pom.xml']
```
The parser and the root pom file should be included, too.